frontend/src/components/processes/ProcessDetailContent.js

```diff
@@ -8,12 +8,13 @@ const {Title} = Typography;
 import AppPageHeader from "../AppPageHeader";
 import PageContent from "../PageContent";
 import ProcessProperties from "../shared/ProcessProperties";
+import ProcessAssociatedMeasurements from "../shared/ProcessAssociatedMeasurements"
 import TrackingFieldsContent from "../TrackingFieldsContent";
-import {listProcesses, listPropertyValues} from "../../modules/experimentRuns/actions";
+import {listProcesses, listPropertyValues, listProcessProperties} from "../../modules/experimentRuns/actions";
 import {download as templateDownload} from "../../modules/importedFiles/actions";
```

Comment:
At runtime I get an error saying `download` does not exist in the importedFiles actions module. Maybe we could remove this import as part of your PR?

Reply:
I feel like it makes more sense to make it a separate PR. And then I merge your PR after your PR is merged.